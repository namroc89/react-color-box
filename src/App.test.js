import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

it("doesn't crash when rendering", () => {
  render(<App />);
});

it("matches snapshot", () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
