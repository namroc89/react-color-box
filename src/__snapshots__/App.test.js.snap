// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <form>
      <label
        for="height"
      >
        Height: 
      </label>
      <input
        id="height"
        name="height"
        type="number"
        value=""
      />
      <label
        for="width"
      >
        Width: 
      </label>
      <input
        id="width"
        name="width"
        type="number"
        value=""
      />
      <label
        for="color"
      >
        Color: 
      </label>
      <input
        id="color"
        name="color"
        type="text"
        value=""
      />
      <button>
        Add a new box!
      </button>
    </form>
    <div
      class="BoxList"
    />
  </div>
</DocumentFragment>
`;
